jellyfin_apiclient_python/api.py

```diff
@@ -617,7 +617,7 @@ def send_request(self, url, path, method="get", timeout=None, headers=None, data
         }
 
         # Changed to use non-Kodi specific setting.
-        if self.config.data.get('auth.ssl') == False:
+        if not self.config.data.get('auth.ssl'):
```

Comment:
SSL defaults to `None` so this change would cause it to be disabled by default.

See:
https://github.com/jellyfin/jellyfin-apiclient-python/blob/e4f0774ebd2922d1c6c2868fdc5792835c4886d1/jellyfin_apiclient_python/configuration.py#L39-L45

Reply:
Hmm. Should we add a default value to the get() that is also False?

Reply:
Maybe, but there is still a chance downstream users are passing none. If you don't do the `==` check there you may want to do it in here.

Reply:
Although I believe I am directly setting it in MPV Shim so others probably are too. As ugly as it is it may be ideal to keep it unchanged.

Reply:
I've changed to using `is` rather than `==`, which should result in the same behaviour and also keeps the linter (that I'll be pushing very soon, stay tuned) happy.